README.md

```diff
@@ -39,13 +39,13 @@ Well, it is easier than you think.
 
 1. Open up your FAVORITE browser. We support way down until Netscape Navigator 7.20 ! (We have tried hard to achieve this.)
 2. Head up to https://devbanner.center
-3. You will be greeted by the beautiful, colorful frontend.
+3. You will be greeted by the new beautiful, colorful frontend by Skayo.
```

Comment:
I'm fine with mentioning your directly for starters, as long as it's okay for you if we might drop that in a couple of weeks or so if the frontend did some major continued evolving.
We'll never drop you from the `Credits` section as long as we use your work tho (Probably not even after dropping all your work, if we should ever do that)

Reply:
That's okay haha